hack/vendor.sh

```diff
@@ -178,4 +178,13 @@ clone git github.com/flynn-archive/go-shlex 3f9db97f856818214da2e1057f8ad8480397
 # metrics
 clone git github.com/docker/go-metrics 86138d05f285fd9737a99bee2d9be30866b59d72
 
+# composefile
+clone git github.com/aanand/compose-file 73f61511606f5b302eda1fe88be1c4d0656fe25b
```

Comment:
License was added to the repo, so will be included if it's revendorded; https://github.com/aanand/compose-file/commit/480a79677acccb83b52c41161c22eaf4358460cc